migrate/index.md

```diff
@@ -1,21 +1,40 @@
 ---
-title: Migrate your data to Tiger Cloud
-excerpt: Migrate your existing database to Tiger Cloud - with downtime, live, with dual-write and backfill, or with livesync
-products: [cloud]
-keywords: [data migration, Postgres, RDS]
-tags: [ingest, migrate, RDS]
+title: Connect, import, ingest, and migrate your data to Tiger Cloud
+excerpt: In Tiger Cloud, you can easily import individual files, ingest data in bulk, migrate from other databases, or connect directly so that your data from another source is continuously synced 
+products: [cloud, self_hosted]
+keywords: [import, ingest, connect, migrate, RDS]
+tags: [import, ingest, connect, migrate, RDS]
 ---
 
 import OpenSupportRequest from "versionContent/_partials/_migrate_open_support_request.mdx"
 
+# Import and connect 
 
-# Migrate and sync data to $CLOUD_LONG
+In $CLOUD_LONG, you can easily add and sync data to your $SERVICE_SHORT from other sources. This includes:
 
-You have chosen to migrate or sync your data to $CLOUD_LONG, thank you. 
+- Connect directly, so data from another source is continuously synced to your $SERVICE_SHORT.
+- Import individual files using $CONSOLE_LONG or the command line.
+- Migrate data from other databases.
+- Ingest metrics from Telegraf.
+
+## Connect to $PG or S3
+
+$CLOUD_LONG provides source connectors for Postgres and S3. You use them to synchronize all or some of your data to your $SERVICE_LONG in real time. You run the connectors continuously, using your data as a primary database and your $SERVICE_LONG as a logical replica. This enables you
+to leverage $CLOUD_LONG’s real-time analytics capabilities on your replica data.
+
+| Connector options                      |  Downtime requirements |
+|----------------------------------------|-----------------------|
+| [$PG_CONNECTOR_CAP][livesync-postgres] | None                  |
+| [$S3_CONNECTOR_CAP][livesync-s3]       | None                  |
+
+
+## Import individual files
+
+You can [import individual files via $CONSOLE_SHORT][import-console], from your local machine or S3. This includes CSV, Parquet, TXT, and MD files. Alternatively, [import data in the terminal][import-terminal].
```

Comment:
using, not via

My mistake, using the terminal. Can you check other mentions of in the terminal while you are at it please.